assets/scss/components/_avatar.scss

```diff
@@ -80,33 +80,33 @@ Class                     | Description
 	color: transparent;
 	display: inline-block;
 	font-size: 0;
-	height: $media-m;
 	text-indent: 100%;
 	white-space: nowrap;
 	vertical-align: top;
-	width: $media-m;
 
-	.inverted & {
-		background: $C_textHintInverted;
+	@include responsiveVarContext--media() {
+		height: $dynamicMedia-m;
```

Comment:
Now that i'm looking at usage, we should probably change this responsive var context mixin name to `responsiveVarContext--scalingMedia` and the vars to `scalingMedia-[size]`.

I can make the update to sasstools before we merge this PR.